### ISSUE TITLE
Delete and Backspace functionality is broken in JAWS 2019 and JAWS 2020

### ISSUE THREAD

**Issue by cwpearsonbw** [FreedomScientific/VFO-standards-support#385]
> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to http://www.google.com
> 2. Shift-Tab back to the About link

In a simple java program with a JTextField within a JPanel, Delete and Backspace functionality is broken in JAWS 2019 and JAWS 2020.
A character is not deleted when Delete is pressed, and Backspace does not announce the deleted character.

### Expected result

> What did you expect to happen?
• When pressing the Backspace key, a single character to the left of the focus is deleted and JAWS 17 announces the deleted character.
• When pressing the Delete key, a single character to the right of the focus is deleted and JAWS 17 announces the next character available.

### Actual result

> What actually did happen?
• When pressing the Backspace key, a single character to the left of the focus is deleted and JAWS 2020 doesn't announce the deleted character.
• When pressing the Delete key, a single character to the right of the focus is not deleted. JAWS 2020 has nothing to announce.

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

Compile and run SwingJPanelDemo at the bottom of this article.
https://www.codejava.net/java-se/swing/jpanel-basic-tutorial-and-examples

#### JAWS version and build number
JAWS 2020.2004.66 - April 2020

 
#### Operating System and version
Windows 10

#### Browser and version:
Not applicable.

**Comment by JeremiahRogersVispero** [comment]
@cwpearsonbw, thanks for posting this issue, and I'm sorry it did not get attention until now. Please note that [JAWS 2026](https://support.freedomscientific.com/Downloads/JAWS/JAWSWhatsNew#latest) includes improved support for Java Swing applications. If the issue still exists in JAWS 2026, can you please provide a working example? Understandably, the link in the original post no longer works. Thanks in advance.
